documentation/src/main/asciidoc/topics/con_near_caches.adoc

```diff
@@ -0,0 +1,35 @@
+[id='near-caches-{context}']
+= Near Caches
+Near caches are local to Hot Rod clients and store recently used data so that every read operation does not need to traverse the network, which significantly increases performance.
+
+Near caches are populated on calls to `get()` or `getVersioned()`.
+When entries are updated or removed in remote caches on {brandname} Servers, those entries are invalidated in the near cache.
```

Comment:
We should probably mention that near cache registers the client listener so it can do this invalidation.